build/source/org/zotero/integration/ooo/comp/Document.java

```diff
@@ -248,13 +248,21 @@ public boolean importDocument() throws Exception {
 				XEnumerationAccess.class, text);
 		XEnumeration xParaEnum = xParaAccess.createEnumeration();
 		ArrayList<XTextContent> removeParagraphs = new ArrayList<XTextContent>();
-		for (int i = 0 ; i < 3 && xParaEnum.hasMoreElements(); i++) {
+		for (int i = 0 ; i < 4 && xParaEnum.hasMoreElements(); i++) {
 			removeParagraphs.add(UnoRuntime.queryInterface(
 					XTextContent.class, xParaEnum.nextElement()));
 		}
 		for (XTextContent paragraph : removeParagraphs) {
 			text.removeTextContent(paragraph);
 		}
+		// remove last paragraph
+		Object lastParagraphObj = null;
+		while(xParaEnum.hasMoreElements()) {
+			lastParagraphObj = xParaEnum.nextElement();
+		}
+		XTextContent lastParagraph = UnoRuntime.queryInterface(XTextContent.class, lastParagraphObj);
+		text.removeTextContent(lastParagraph);
+		// return
```

Comment:
I know the import-export workflow inside LibreOffice creates a trailing paragraph, but this is not true for other word processors, and generally removing the final paragraph with other document processors can throw errors in certain cases, so the code just allows the trailing paragraph to stay as that's a lesser evil.